api/src/pcapi/scheduled_tasks/commands.py

```diff
@@ -191,13 +191,26 @@ def send_email_reminder_7_days_before_event() -> None:
         send_reminder_7_days_before_event_to_pro(stock)
 
 
-@blueprint.cli.command("send_email_reminder_tomorrow_event_to_beneficiaries")
-@log_cron_with_transaction
 def send_email_reminder_tomorrow_event_to_beneficiaries() -> None:
     """Triggers email reminder to beneficiaries for none digitals events happening tomorrow"""
     individual_bookings = bookings_repository.find_individual_bookings_event_happening_tomorrow_query()
     for individual_booking in individual_bookings:
-        send_individual_booking_event_reminder_email_to_beneficiary(individual_booking)
+        try:
+            send_individual_booking_event_reminder_email_to_beneficiary(individual_booking)
+        except Exception:  # pylint: disable=broad-except
+            logger.exception(
+                "Could not send email reminder tomorrow event to beneficiaries",
```

Comment:
(typo)

```suggestion
                "Could not send email reminder tomorrow event to beneficiary",
```

... puisque l'erreur n'est (a priori !) survenue que sur une réservation, donc un seul bénéficiaire.